syntax = "proto3";
package com.echo.proto;

message EchoRequest {
  string message = 1;
}

message EchoResponse {
  string message = 1;
}

service EchoService {
  rpc echo (EchoRequest) returns (EchoResponse);
}
